Wikipedia/Code/ArticleContextMenuPresenting.swift

```diff
@@ -74,7 +74,7 @@ extension ArticleContextMenuPresenting {
             let config = UIContextMenuConfiguration(identifier: linkURL as NSURL, previewProvider: { () -> UIViewController? in
                 return peekParentVC
             }) { (suggestedActions) -> UIMenu? in
-                return previewMenu
+                return (peekParentVC as? ArticleContextMenuPresenting)?.previewMenu
```

Comment:
@mcleinman it seems like this is all that was needed to fix my issues 1 and 2 from my last review. Given that, maybe we should revert your changes you made in the `readMoreArticlePreviewActionSelected` methods in [this commit](https://github.com/wikimedia/wikipedia-ios/pull/4094/commits/c8fc4af1acbc7cdd18956911ec12b2ad367e05f7). I also have one misgiving about your navigate method (see below), so this would take care of looking into that.

Reply:
Hmmm, seems so. I don't remember what led me to also work on the navigation changes. I'm going to back out my other changes.

Peek/pop will still use different navigation from the rest of the app. If we see any unintended behavior, this is likely why. But it should rightfully fixed in a separate PR.

Reply:
Sounds good, we'll keep an eye out. Thanks!